### ISSUE TITLE
'\n' characters in sentences in *.json data files.

### ISSUE THREAD

**Issue by j-min** [airsplay/lxmert#62]
There are some sentences having `\n` characters.
Ex) There are 147 `\n`s in `mscoco_minival.json` file as below.

![image](https://user-images.githubusercontent.com/18069263/82767172-5e5ff600-9dda-11ea-8584-6224ac0d0dbb.png)
![image](https://user-images.githubusercontent.com/18069263/82767100-b21e0f80-9dd9-11ea-901f-df0d64630f8f.png)
![image](https://user-images.githubusercontent.com/18069263/82767114-c3671c00-9dd9-11ea-8be3-e35653d9bfc0.png)

**Comment by j-min** [comment]
Not only `\n`. There are many other cases like this, which adds "." sentences during training.
Is there a code somewhere that cleans these weird sentences?

![image](https://user-images.githubusercontent.com/18069263/82767212-bd256f80-9dda-11ea-9a56-7b57dc1606cf.png)

**Comment by airsplay** [comment]
This is the raw annotations from the MS COCO dataset. Details could be found in the `annotations` folder after downloading the dataset.

**Comment by j-min** [comment]
Yes, I just found same problems in the raw COCO annotations. Similar problems are in other datasets as well (VG also have similar issue in the second image).
Is this handled somewhere in your code?

**Comment by airsplay** [comment]
Yes... I also just realized it recently... This code is build based on the BERT tokenizer,  which will consider \n's as spaces. So it does not require explicit handler. (That's why I just notice this when using some other tokenizers...)
